infra/README.md

```diff
@@ -202,6 +211,56 @@ tags = merge(module.common.default_tags, {
 
 })
 ```
+
+## Destroying Infrastructure
+
+To destroy everything you'll need to undo everythin in reverse.
+
+1. First destroy all your environments by running `terraform destroy` in each of your environment module folders
+
+    ```bash
+    # within each env folder in /infra/envs/ (dev, stage, etc)
+    terraform destroy
+    ```
+
+2. Then to destroy the backends, first you'll need to add `force_destroy = true` to the S3 buckets, and update the
```

Comment:
Should this have more information somewhere on why we need to do this? e.g. indicating that S3 buckets are protected from destruction by default to avoid loss-of-data, and these changes will force the bucket and all contents to be permanently destroyed?

Reply:
Sounds good, added an explanation